src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimIssueReceiptService.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
```

Comment:
can we have all receipts service in its own package uk.gov.hmcts.cmc.claimstore.document.receipts

Reply:
@geek-soft Thoughts?